sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexClientBuilder.java

```diff
@@ -57,16 +109,54 @@
  * </pre>
  * <!-- end com.azure.search.documents.indexes.SearchIndexAsyncClient.instantiation -->
  *
- * <p><strong>Instantiating a synchronous Search Index Client</strong></p>
  *
- * <!-- src_embed com.azure.search.documents.indexes.SearchIndexClient.instantiation -->
+ * <h4>
+ *     Building and Authenticating a {@link SearchIndexClient} or {@link SearchIndexAsyncClient} using Azure Active Directory
```

Comment:
```suggestion
 *     Building and Authenticating a {@link SearchIndexClient} or {@link SearchIndexAsyncClient} using Microsoft Entra
```

Right? Shouldn't all Azure Active Directory references now be Microsoft Entra?